config/locales/en.yml

```diff
@@ -34,6 +34,7 @@ en:
       cancel_button: Cancel
       empty: No ToDo lists have been added yet.
       invitation: You can create your first one right now.
+    create_info_html: Created by <span>%{user}</span> on %{date}
```

Comment:
@vlasiak, Here I recommend to avoid using of html, use just yml abilities. Please move out html into the view and rename the key please.